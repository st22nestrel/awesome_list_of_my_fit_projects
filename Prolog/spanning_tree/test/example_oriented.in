A B
B A
A C
B C