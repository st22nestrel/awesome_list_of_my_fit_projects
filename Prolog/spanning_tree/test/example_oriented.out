A-B A-C 
A-B B-C 
B-A A-C 
B-A B-C 
A-C B-C 
